snet_cli/mpe_service_command.py

```diff
@@ -171,6 +174,40 @@ def print_service_metadata_from_registry(self):
         metadata      = self._get_service_metadata_from_registry()
         self._printout(metadata.get_json_pretty())
 
+    def _service_status(self, url, secure=True):
+        channel = None
+        try:
+            channel = open_grpc_channel(endpoint=url)
+            stub = heartb_pb2_grpc.HealthStub(channel)
+            try:
+                response = stub.Check(heartb_pb2.HealthCheckRequest(service=""), timeout=10)
+            except grpc.RpcError as err:
+                err_code = str(err.code())
+                print(err_code)
```

Comment:
please use ```self._printerr``` or ```self._printout``` instead of `print`. By the way, do you really want to print something or it was a debug message?
Because, If you don't want to print this error messages then you don't need this complicated nested try. You could simply write:
```python
        try:
            channel = open_grpc_channel(endpoint=url)
            stub = heartb_pb2_grpc.HealthStub(channel)
            response = stub.Check(heartb_pb2.HealthCheckRequest(service=""), timeout=10)
            if response!=None and response.status == 1:
                return True
        except Exception as e:
                return False
```